packages/flutter/lib/src/widgets/animated_scroll_view.dart

```diff
@@ -85,6 +85,92 @@ class AnimatedList extends _AnimatedScrollView {
     super.clipBehavior = Clip.hardEdge,
   }) : assert(initialItemCount >= 0);
 
+  /// A scrolling container that animates items with separators when they are inserted or removed.
+  ///
+  /// This widget's [AnimatedListState] can be used to dynamically insert or
+  /// remove items. To refer to the [AnimatedListState] either provide a
+  /// [GlobalKey] or use the static [of] method from an item's input callback.
+  ///
+  /// This widget is similar to one created by [ListView.separated].
+  ///
+  /// {@tool dartpad}
+  /// This sample application uses an [AnimatedList.separated] to create an effect when
+  /// items are removed or added to the list.
+  ///
+  /// ** See code in examples/api/lib/widgets/animated_list/animated_list_separated.0.dart **
+  /// {@end-tool}
+  ///
+  /// By default, [AnimatedList.separated] will automatically pad the limits of the
+  /// list's scrollable to avoid partial obstructions indicated by
+  /// [MediaQuery]'s padding. To avoid this behavior, override with a
+  /// zero [padding] property.
```

Comment:
Should 'zero' here be `[EdgeInsets.zero]`?

Reply:
Not sure, I copy pasted this from [AnimatedList]'s docstring :)

Reply:
Hmm. We should probably know what these docs mean before we commit this.

Reply:
Depending on the use case one might want to remove only the padding for a specific side. So IMO "override with a zero [padding] property." is more fitting than "override with `EdgeInsets.zero` [padding] property."

What do you think?

Reply:
SGTM! Thanks for verifying!